extensions/example-extension/page.tsx

```diff
@@ -63,3 +63,77 @@ export class ExamplePage extends React.Component<ExamplePageProps> {
     );
   }
 }
+
+@observer
+export class SimplePage extends React.Component<{ extension: LensRendererExtension }> {
+  render() {
+
+    return (
+      <div className="flex column gaps align-flex-start" style={{ padding: 24 }}>
+        <div style={{ width: 200 }}>
+          <CoffeeDoodle accent="#3d90ce"/>
+        </div>
+
+        <div>Hello from Example extension!</div>
+        <div>Location: <i>{location.href}</i></div>
```

Comment:
`location.href` is non-observable (if you want to see any changes related to params state).
`@observer` also not required in current example.